### ISSUE TITLE
What's going on with this project?

### ISSUE THREAD

**Issue by AlexanderChen1989** [google/novm#41]
No more commits.

**Comment by vtolstov** [comment]
can someone from google team share info about feature of this project?

**Comment by resouer** [comment]
@vtolstov They are doing this project internally. But you can subscribe [KataContainers](https://katacontainers.io) which should have big news next month related to this project.

**Comment by vtolstov** [comment]
@resouer thanks, but i don't understand how novm related to kotacontainers?

**Comment by resouer** [comment]
@vtolstov they are all hypervisor based container and have cooperation.

**Comment by vtolstov** [comment]
Yes, but kota containers have qemu integrated, and i need pure kvm...

**Comment by amscanne** [comment]
It's unlikely that there will be updates to this project soon.

I am confused about the KataContainers pitch. This specific project has no formal connection to KataContainers nor cooperation that I am aware of. Big news with respect to this project would be news to me -- I assume this means container and virtualization news generally.

**Comment by resouer** [comment]
@amscanne Emm ... sorry for misleading, I am not referring to this specific project since novm has been inactive for years. I should have say: "big news and cooperation related to hypervisor based container". But AFIK, they still related to this project technically.
